cds-snc/cppd-medical-report
# Fixing names on treatment and medication buttons

# Description

Adds English to the "Add treatment" and "Add medication" buttons 

# Test Instructions

1. Navigate to */en/add_treatment and */en/add_medication
2. Verify that the buttons say "Add treatment" and "Add medication", respectively 

# Notes
Fixes bugs in comments on these tickets: 
https://trello.com/c/JuoS9in0/335-physician-can-input-information-on-other-treatments
https://trello.com/c/OLYBv8uG/334-physician-can-input-information-on-their-patients-medications
